app/models/concerns/vacancy_job_summary_validations.rb

```diff
@@ -11,10 +11,10 @@ def about_school_must_not_be_blank
     # Since vacancy is set by VacancyForm.initialize, it can be undefined here.
     if defined?(vacancy) && vacancy&.job_location == 'central_office'
       organisation = 'trust'
-    elsif defined?(vacancy)
+    elsif defined?(vacancy) && (vacancy&.job_location == 'at_one_school' || vacancy&.job_location == nil)
       organisation = 'school'
     else
```

Comment:
`job_location` should never be nil, so this else statement should be unnecessary